packages/blade/src/components/Checkbox/__tests__/__snapshots__/CheckboxGroup.native.test.tsx.snap

```diff
@@ -16,12 +16,12 @@ exports[`<CheckboxGroup /> should render with label 1`] = `
 >
   <View
     marginBottom="spacing.2"
-    marginRight="spacing.5"
+    marginRight="spacing.0"
     style={
       Array [
         Object {
           "marginBottom": 4,
-          "marginRight": 16,
+          "marginRight": 0,
```

Comment:
Label is always on top on native but for some reason native had a marginRight of 16px. Removed that now.